expectise/expect.py

```diff
@@ -245,8 +262,9 @@ def tear_down(cls) -> None:
             remain = args["expected"] - args["performed"]
             if remain > 0:
                 message += f"`{method_name}` from class `{class_name}` still expected to be called {remain} time(s).\n"
-            # Resetting Expect parameters and original methods
-            cls.set_up(cls.class_h[class_name], method_name, args["method"])
+            # Resetting Expect parameters and original methods, keeping the reference to the Mock object
+            curr_mock = cls.method_h[(class_name, method_name)]["mock"]
+            cls.set_up(curr_mock)
```

Comment:
tiny comment: no need to store `curr_mock`?

Reply:
Good point, removed. Also I can't do the merge, it's blocked for me :) so go ahead and press the green button whenever

Reply:
oh sorry about that, still have to figure out the proper settings 😬